scanpy/tests/external/test_scrublet.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import scanpy as sc
+import scanpy.external as sce
+
+pytest.importorskip("scrublet")
+
+
+def test_scrublet():
+    """
+    Test that Scrublet run works.
+
+    Check that scrublet runs and detects some doublets.
+    """
+    adata = sc.datasets.pbmc3k()
+    sce.pp.scrublet(adata)
+
+    errors = []
+
+    # replace assertions by conditions
+    if not set(['predicted_doublet', 'doublet_score']).issubset(adata.obs.columns):
+        errors.append("doublet columns not set in .obs")
+
+    if len(adata.obs_names[adata.obs.predicted_doublet]) == 0:
+        errors.append("no doublets identified")
```

Comment:
```suggestion
    assert adata.obs["predicted_doublet"].any(), "Expect some doublets to be identified"
```

This is boolean valued, right?

Reply:
yup